src/frontend/sandbox/cunningham.ts

```diff
@@ -0,0 +1,10 @@
+// cunningham.ts
+import { DefaultTokens } from '@openfun/cunningham-react';
+
```

Comment:
It's the right place to drop a link to explain how cunningham customization works : )

```suggestion
/*
  You can redefine here all Cunningham tokens to fit look'n feel to your needs.
  Take a look at https://openfun.github.io/cunningham/?path=/docs/getting-started-customization--docs
  for further explanation about customization.
*/
```